docs/docs/quickstart.md

```diff
@@ -99,16 +103,15 @@ Congratulations - you've written your first Kurtosis code!
 
 ### Review: Hello, World
 :::info
-We'll use these "Review" sections to explain what happened in the section. If you just want the action, feel free to skip them.
+You'll use these "Review" sections to explain what happened in the section. 
 :::
 
-In this section, we created a `main.star` file that prints `Hello, world`. The `.star` extension corresponds to [the Starlark language developed at Google][starlark-github-repo], a dialect of Python for configuring the [Bazel build system][bazel-github]. [Kurtosis uses Starlark for the same purpose of configuring builds][starlark-explanation], except that we're building a distributed application rather than binaries or JARs.
+In this section, you created a `main.star` file that simply told Kurtosis to print `Hello, world`. The `.star` extension corresponds to [Starlark][starlark-explanation], which is also used by Google and Meta as a build system configuration language.
```

Comment:
```suggestion
In this section, you created a `main.star` file that simply told Kurtosis to print `Hello, world`. **The `.star` extension corresponds to [Starlark][starlark-explanation], a Python dialect also used by Google and Meta for configuring build systems.**
```

Reply:
bolded this because people keep overlooking it, and changed the wording a bit

Reply:
Spoke to Galen offline, not going to bold this section.